### PR TITLE
Ensures that the version numbers for cdnjs are the most recent

### PR DESCRIPTION
**Pull request by ColinEberhardt**
Assemble was missing the string manipulation functionality I required, so I added `slice` to handlebars helpers:

https://github.com/assemble/handlebars-helpers/pull/194

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "grunt-contrib-watch": "^0.6.1",
     "grunt-jscs": "^1.8.0",
     "grunt-version": "^1.0.0",
+    "handlebars-helpers": "git://github.com/ColinEberhardt/handlebars-helpers.git",
```
Comment (chrisprice) [inline package.json]: I think you can shorten this to just ColinEberhardt/handlebars-helpers

**Comment by chrisprice** [comment]
One comment but otherwise :+1:

File: package.json (hunk 1 above)
Comment (ColinEberhardt) [inline package.json]: > I think you can shorten this to just ColinEberhardt/handlebars-helpers

Yep - that worked.
